wallet/res/values/strings-extra.xml

```diff
@@ -242,6 +242,25 @@
     <string name="block_info">Block Info</string>
     <string name="confirm">Confirm</string>
 
+    <string name="auto_logout">Auto Logout</string>
+    <string name="spending_confirmation">Spending Confirmation</string>
+    <string name="logout_after">Logout after</string>
+    <string name="immediately">Immediately</string>
+    <string name="twenty_four_hours">24 hours</string>
+    <string name="one_minute">1 minute</string>
+    <string name="five_minute">5 minute</string>
+    <string name="one_hour">1 hour</string>
+    <string name="one_day">1 day</string>
+    <string name="biometric_limit">Biometric Limit</string>
+    <string name="spending_confirmation_hint_zero">PIN is always required to make a payment</string>
+    <string name="spending_confirmation_hint_above_zero">You can authenticate with Touch ID for payments below</string>
```

Comment:
The wording "Touch ID" is specific of iOS, on Android we use "Fingerprint Authentication" or "Biometric Authentication".